File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md

```diff
@@ -0,0 +1,267 @@
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Access the section in a Word document.
+IWSection section = document.Sections[0];
+//Add a new paragraph to the section.
+IWParagraph paragraph = section.AddParagraph();
+paragraph.ParagraphFormat.FirstLineIndent = 36;
+paragraph.BreakCharacterFormat.FontSize = 12f;
+IWTextRange text = paragraph.AppendText("In 2000, Adventure Works Cycles bought a small manufacturing plant, Importadores Neptuno, located in Mexico. Importadores Neptuno manufactures several critical subcomponents for the Adventure Works Cycles product line. These subcomponents are shipped to the Bothell location for final product assembly. In 2001, Importadores Neptuno, became the sole manufacturer and distributor of the touring bicycle product group.");
+text.CharacterFormat.FontSize = 12f;
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 9: Add below code example to **save the Word document in WinUI Desktop app**.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Save a Word document to the stream.
+using MemoryStream outputStream = new();
+document.Save(outputStream, FormatType.Docx);
+//Save the stream as a Word document file in the local machine.
+Save(outputStream, "Sample.docx");
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Read-and-Save-document/Open-and-save-Word-document/WinUI/WinUI-Desktop-app).
+
+By executing the program, you will get the **Word document** as follows.
+
+![WinUI Desktop open and save output Word document](WinUI_Images/OpenAndSaveOutput.png)
+
+## WinUI UWP app
+
+Step 1: Create a new C# WinUI UWP app. Select Blank App (WinUI 3 in UWP)from the template and **click** the Next button.
+
+N> To get the UWP Experimental project templates and build UWP apps with WinUI 3, you should download the [Windows App SDK Experimental Extension](https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftProjectReunionPreview) for Visual Studio.
+
+Step 2: Enter the project name and click **Create**.
+
+Step 3: Set the Target version to Windows 10, version 2004 (build 19041) and the Minimum version to Windows 10, version 1809 (build 17763) and then click **OK**.
+
+Step 4: Install the [Syncfusion.DocIO.NET](https://www.nuget.org/packages/Syncfusion.DocIO.Net) NuGet package as a reference to your project from the [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering a Syncfusion license key in your application to use our components.
+
+Step 5: Add a new button in the **MainPage.xaml** as shown below.
+
+{% tabs %}
+
+{% highlight XML %}
+
+<Page
```

Comment:
<Page
    x:Class="OpenAndSave.MainPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:OpenAndSave1"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d"
    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">

    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
        <Button x:Name="button" Click="OpenAndSaveDocument">Open and save Word document</Button>
    </StackPanel>
</Page>